.travis.yml

```diff
@@ -0,0 +1,20 @@
+sudo: required
+services:
+  - docker
+matrix:
+  include:
+    - language: node_js
+      node_js:
+        - "10"
+      cache:
+        yarn: true
+        directories:
+          - "node_modules"
```

Comment:
We also need to cache `packages/react-native-oursky/node_modules` and `packages/react-native-oursky-example/node_modules`